app/src/main/res/values/fonts.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="font_roboto_medium">sans-serif-medium</string>
```

Comment:
I really like this! I always find it difficult to map UX specifications like `robotium_modium` to `san-serif-medium` – good work! :)

Reply:
Thanks